wpcsite/settings.py

```diff
@@ -32,7 +32,7 @@
     BROKER_URL is undefined, or 0 if defined
 CELERY_RESULT_BACKEND - Backend URL string for Celery
 DATABASE_URL - See https://github.com/kennethreitz/dj-database-url
-DEFAULT_FROM_EMAIL - "From" email for emails to users
+DEFAULT_FROM_EMAIL - 'From' email for emails to users
```

Comment:
I think in prose, I'd stick with " rather than '. I don't think it matters a whole lot, though.